models/quickbooks__general_ledger_by_period.sql

```diff
@@ -10,7 +10,19 @@ retained_earnings as (
     from {{ ref('int_quickbooks__retained_earnings') }}
 ),
 
-final as (
+ordinals as (
+
+    select 
+        cast(account_class as {{ dbt.type_string() }}) as account_class,
+        cast(account_type as {{ dbt.type_string() }}) as account_type,
+        cast(account_sub_type as {{ dbt.type_string() }}) as account_sub_type,
+        cast(account_number as {{ dbt.type_string() }}) as account_number,
+        ordinal
+    from {{ var('financial_statement_ordinal') }}
+),
+
+
```

Comment:
My thoughts that were not posted for some reason:
```sql
{% if var('financial_statement_ordinal') %}
ordinals as (

    select 
        cast(account_class as {{ dbt.type_string() }}) as account_class,
        cast(account_type as {{ dbt.type_string() }}) as account_type,
        cast(account_sub_type as {{ dbt.type_string() }}) as account_sub_type,
        cast(account_number as {{ dbt.type_string() }}) as account_number,
        ordinal
    from {{ var('financial_statement_ordinal') }}
),
{% endif %}
```

Reply:
Updated!